modules/react/text-input/lib/InputIconContainer.tsx

```diff
@@ -19,6 +19,7 @@ const IconContainer = styled('div')({
   right: space.xxs,
 });
 
+/** @deprecated Use `InputGroup` instead. */
```

Comment:
Remind me to put something about this in the release guide, so My doesn't have a surprise. 😄

Reply:
I'll deprecate in 9.1 instead and not change v9 comms